### DISCUSSION TITLE
Move asdf-flutter to here

### DISCUSSION THREAD

**Discussion by oae** [asdf-community/.github#56] [General]
I would like to pass maintainership of the `asdf-flutter` plugin to `asdf-community`.

**Comment by smorimoto** [comment]
Consult this discussion post: https://github.com/orgs/asdf-community/discussions/57 This is the smoothest way to go. Of course I can do all the work for you, but it's done on a best effort basis. (I've been busy with work lately...)

**Reply by NotHolst** [reply]
@oae I would like to nudge this.
If you want you can transfer the repo to me and i can handle the transfer process?

**Comment by ken-ty** [comment]
This repository has been migrated for my account.
I continue handle the transfer process.

**Reply by ken-ty** [reply]
https://github.com/asdf-community/infrastructure/pull/141

**Reply by ken-ty** [reply]
https://github.com/asdf-community/infrastructure/pull/157

**Reply by ken-ty** [reply]
Once https://github.com/asdf-community/infrastructure/pull/157 is successfully merged, the migration is complete.

**Reply by ken-ty** [reply]
https://github.com/asdf-community/infrastructure/pull/157 is successfully merged, thank you everyone!
